### DISCUSSION TITLE
PDF Layers

### DISCUSSION THREAD

**Discussion by Terrox** [farling42/fvtt-pdf-pager#56] [Q&A] [answered]
Can I toggle PDF layers upon clicking an element? 
I'm thinking of Clocks during get/set value and updating the segments. 
Also fancy radio buttons, checkboxes etc.
example 
![image](https://user-images.githubusercontent.com/16533005/229454350-544178ad-8263-463a-909f-9ed24039c5c2.png)

**Comment by farling42** [comment, marked as answer]
PDF Pager uses the underlying pdfjs library which is supplied with Foundry V10.

If this sort of capability works in a normal Foundry journal PDF page, then it should work in the PDF Pager module too.
